Tests/Marketplace/zip_packs.py

```diff
@@ -229,10 +277,6 @@ def main():
             os.mkdir(zip_path)
         artifacts_path = '/home/runner/work/content-private/content-private/content/artifacts'
```

Comment:
Please expose a script var for artifacts folder.

Reply:
Do you mean a variable that can be set from the step call in config.yml?
Because it seems that there's already one, you can see it in line 251.

Reply:
yes, no need to hard-code the artifacts path, just pass it to the script

Reply:
Ok I deleted this hard-coded value and kept the definition where it gets the value from the config.yml. The $CIRCLE_ARTIFACTS should contain the right value either when it's private build or not?